intelmq/lib/test.py

```diff
@@ -146,6 +146,7 @@ def setUpClass(cls):
         cls.use_cache = False
         cls.allowed_warning_count = 0
         cls.allowed_error_count = 0  # allows dumping of some lines
+        cls.allow_checks: bool = False
```

Comment:
The previous behaviour was to test the checks too. Why do we want to not test them by default?

Reply:
Renamed for better clarity. Default behavior wasn't really changed. This was ment to allow testing of checks to be skipped.